README.md

```diff
@@ -24,15 +24,10 @@ git clone https://github.com/MUNComputerScienceSociety/Website
 # Enter cloned folder
 cd Website
 
-# Compile
-lektor build
-# To specify which folder to build to, append
-lektor build --output-path folder-name
-
-# Spin up a webserver in the outputs directory, for example:
-cd /lektor/build/output/folder-name
-python -m SimpleHTTPServer
-# Then open http://localhost:8000 in a web browser
+# Launch a Lektor server to auto-compile and host the site
+lektor server
+
+# Now go to localhost:5000
```

Comment:
I think we should suffix this with `in a web browser`.

Reply:
And prefix localhost:5000 with `http://`

Reply:
Makes sense. I'll add it sometime tonight

Reply:
👍